### PR TITLE
CI: Add support for PHP8.2

### PR DESCRIPTION
**Pull request by shulard**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,14 +6,20 @@ documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [6.2.0] - 2022-01-24
+
+- Support PHP 8.1
```
Comment (jaylinski) [inline CHANGELOG.md:11]: \* 8.2

**Review by jaylinski — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (shulard) [reply CHANGELOG.md]: This was a rebase mistake, this change is from the "prepare 6.2.0 release" commit.

**Review by jaylinski — APPROVED** [review]

**Comment by jaylinski** [comment]
Depends on https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues?q=is%3Aissue+is%3Aopen+8.2+label%3Atopic%2FPHP8.2.

**Comment by shulard** [comment]
> Depends on PHP 8.2 support in php-cs-fixer: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues?q=is%3Aissue+is%3Aopen+8.2+label%3Atopic%2FPHP8.2.

Ok I've applied a tiny suggestion from the php-cs-fixer CLI. It tells to add `PHP_CS_FIXER_IGNORE_ENV` environment variable to bypass the compatibility warning. Since the PHP-CS-Fixer compat is only related to some of the sniffs, I thing it's pretty safe on our side since we have a small amount of code to manage…

**Review by jaylinski — APPROVED** [review]

**Pull request merged**